pkg/ui/package.json

```diff
@@ -88,7 +88,7 @@
     "bytebuffer": "^5.0.1",
     "cache-loader": "^4.1.0",
     "chai": "^4.1.0",
-    "copy-webpack-plugin": "^4.5.1",
+    "copy-webpack-plugin": "^6.1.0",
```

Comment:
@dhartunian , the API is changed with a major upgrade for `copy-webpack-plugin` library.
Now constructor accepts an object with `patterns` field with an array of source/destination paths. 
The following change has to be made in `webpack.app.js` file to avoid build failure.
(`webpack.app.js` line 170)
``` 
new CopyWebpackPlugin({
   patterns: [{ from: "favicon.ico", to: "favicon.ico" }]
}),
```